pkg/crc/network/httpproxy/proxy.go

```
@@ -41,7 +42,7 @@ func (p *ProxyConfig) String() string {
 }
 
 func readProxyCAData(proxyCAFile string) (string, error) {
-	if proxyCAFile == "" {
+	if proxyCAFile == "" || !filepath.IsAbs(proxyCAFile) {
```

I'm not sure why we want to ignore the error in this situation?